src/components/Actions/CardLabel.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export type CardLabelProps = {
+  title: string;
+  value?: React.ReactNode;
```

Comment:
For the title or the value? Seems like they have equal weighting so I'm thinking maybe we shouldn't use children?

Reply:
Ya, fair point. I was thinking for the `value`, since it always seems like a bit of a smell to have a `ReactNode` prop if you're not already using `children` to render content.

Reply:
done